addons/udes_stock_routing/models/stock_move.py

```diff
@@ -337,55 +336,40 @@ def _do_unreserve(self):
                     continue
                 split_pick_type = rule.picking_type_id
                 original_picking_type = rule.u_run_on_assign_applicable_to
+                if split_pick_type == original_picking_type:
+                    _logger.debug(
+                        "Skipping because split and original picking types are the same: %r",
+                        split_pick_type | original_picking_type,
```

Comment:
Do we need the | here if they are the same?

Reply:
No fixed in ab1a7ef3c